### ISSUE TITLE
Design notes

### ISSUE THREAD

**Issue by tuckabelle** [CSIS-iLab/russia-sanctions-tracker#1]
**General type**
- [x] Change sans-serif to Source Sans Pro (and reset letter spacing)
- [x] Add simple loading icon as table loads
- [x] the white text on light blue background doesn’t have a high contrast ratio, so it could be really difficult for people to see. Change to the dark blue

**Search Box**
- [ ] Remove 'Show/Hide Sanctions' button. People are here to see the table, so that should be visible ASAP
- [x] add input IDs and label for `<label for="my-input">Label text here...</label><input id="my-input">` 

**Table**
- [x] allow table to go wider than the rest of the page, max-width = 1500px
- [x] change header `<th>` font to source sans pro, font-size: 14px; letter-spacing: 1px;
- [x] Remove extra top padding for first data row cells (`table tr:first-of-type td`)
- [x] add scope=“col” to th column headers
- [x] reduce border spacing `table.dataTable {border-spacing: 6px;
- [x] odd table row color: #f1f1f1
- [x] table row hover color: #E1E4EA
- [ ] put the sorting column (date) as first column
- [x] sort arrow is hard to see… make it white, and either align it with the “date” label or align it under/centered 
- [x] sort arrow disappears when you click it to reverse direction

**Expand**
- [x] Change expand icons to the ones I sent you :)
- [x] Remove the "Expand" word that appears on the hover, and keep the icon the same size as the table grows/shrinks
- [x] In the paragraph at the top, please include the expand symbol in the description "Click the [icon] symbol to view addition information and resources about an individual sanction,."

**Details box**
- [x] For the details box, align the details under the second column (now  there should be an empty `<tr>` and the `<tr>` with content should have a `colspan=“7”`)
- [x] Add `scope="colgroup"` to the `<tr>` with colspan
- [x] headers should be source sans pro (font weight 400, letter spacing .025rem, )
- [x] info should be expo serif (  font-size: 17px;  line-height: 1.4;)
- [ ] make the bulleted items a list `<ul><li>` (right now each is in a separate table cell)
- [x] line up the “resources” header with and the headers above it

**Functionality**
- [x] the table auomatically updates when you type, so it should do the same if someone deletes text
- [ ] why can you only choose either a “targeted individuals” or “targeted companies”?

**Table footer** 
- [x] *Showing* [how many listings are displayed] *of* [total number of listings] *sanctions*
- [x] Include the "Showing..." phrase in the  Search box as welL! (Centered, under the form)

**Images**
- [x] add alt attribute to images
